docs/syscall_descriptions_syntax.md

```diff
@@ -167,6 +167,31 @@ accept(fd sock, ...) sock
 listen(fd sock, backlog int32)
 ```
 
+Resources don't have to be necessarily returned by a syscall. They can be used as any other data type. For example:
+
+```
+resource fd_request[fd]
+
+ioctl$MEDIA_IOC_REQUEST_ALLOC(fd fd_media, cmd const[MEDIA_IOC_REQUEST_ALLOC], arg ptr[out, fd_request])
+ioctl$VIDIOC_QBUF(fd fd_video, cmd const[VIDIOC_QBUF], arg ptr[inout, v4l2_buffer])
+
+v4l2_buffer {
+	index		int32
```

Comment:
Please shorten this example to capture only the necessary parts.
It contains lots of irrelevant for the point details and it may hard to understand for a new comer what we actually trying to show here and what's important and what's not.